src/xdocs/config_coding.xml

```diff
@@ -4468,6 +4468,31 @@ enum Foo10 { FOO, BAR } // OK
         <source>
 &lt;module name=&quot;NoFinalizer&quot;/&gt;
         </source>
+        <p>Example:</p>
+          <source>
```

Comment:
Please fix indentation. Should be the same level as `<p>`

Reply:
done